tests/user_commands/management/commands/mutually_exclusive_required_with_boolean.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        group = parser.add_mutually_exclusive_group(required=True)
+        group.add_argument('--flag_true', action='store_true')
+        group.add_argument('--flag_false', action='store_false')
+        group.add_argument('--const', action='store_const', const=42)
```

Comment:
Have you checked other [actions](https://docs.python.org/3/library/argparse.html?#action)? e.g. `append_const` and `count`?